<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameRanker</class>
 <widget class="QMainWindow" name="GameRanker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLoadData">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGame1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Game 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leUsername">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BGG Username:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Rating:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMinRating">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGame2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Game2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdCollection">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="taCollection">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>491</width>
      <height>830</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="taTop">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>491</width>
      <height>831</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column/>
    <column/>
   </widget>
   <widget class="QPlainTextEdit" name="txtGame1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtGame2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leUsername</tabstop>
  <tabstop>leMinRating</tabstop>
  <tabstop>btnLoadData</tabstop>
  <tabstop>btnGame1</tabstop>
  <tabstop>btnGame2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
